p/themes/base-theme/template.css

```diff
@@ -412,6 +494,9 @@ a.btn {
 	white-space: nowrap;
 	text-overflow: ellipsis;
 }
+.tree-folder:last-of-type {
+	margin-bottom: 15em;
```

Comment:
That does not work when the last category is invisible 😢 such as when all articles are read.

Reply:
Fixing

Reply:
Second attempt in https://github.com/FreshRSS/FreshRSS/pull/2117/commits/10cb067e11cebf79a185f05dfbc4a59caa3f7204